### DISCUSSION TITLE
single site only with apache?

### DISCUSSION THREAD

**Discussion by gekko2725** [OpenIDC/mod_auth_openidc#817] [Q&A] [answered]
Hi folks,

I have just started using openidc and I am really liking it.  I have a quick question on implementation for apache:

Q. Is it possible to implement the mod (/etc/apache2/mods-available/auth_openidc.conf) so that it only applies to one apache site?

I have several virtual hosts on the same server and right now my config seems to be encompassing all of them.

Any tips, links or reading material would be greatly appreciated.

Thanks!

**Comment by zandbelt** [comment, marked as answer]
Just include auth_openidc.conf in the appropriate 'VirtualHost' config and use AuthType openid-connect in its locations, and only there.

**Reply by gekko2725** [reply]
Makes sense, thanks!
